setup.py

```diff
@@ -24,26 +25,35 @@
     "Programming Language :: Python :: 3.7",
     "Programming Language :: Python :: 3.8",
     "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
     "Topic :: Scientific/Engineering :: Mathematics"
 ]
 
-extensions = [Extension('quadprog', [
-    'quadprog/linear-algebra.c',
-    'quadprog/qr-update.c',
-    'quadprog/quadprog.c',
-    'quadprog/solve.QP.c',
-])]
+ext_modules=cythonize(
+    [
+        Extension(
+            "quadprog",
+            [
+                'quadprog/linear-algebra.c',
+                'quadprog/qr-update.c',
+                'quadprog/solve.QP.c',
+                'quadprog/quadprog.pyx',
+            ],
+        )
+    ],
+)
 
 setup(
-    name="quadprog",
+    name="quadprog-wheel",
     version="0.1.11",
-    description="Quadratic Programming Solver",
+    description="Quadratic Programming Solver with wheel packages",
```

Comment:
This is the description for https://pypi.org/project/quadprog-wheel/, can also be reverted to the original for merging.